src/shared/config/apiUrls.ts

```diff
@@ -494,10 +494,37 @@ export const getDownloadUrl = ({
 
 const proteinsApiPrefix = 'https://www.ebi.ac.uk/proteins/api';
 export const proteinsApi = {
+  coordinates: (accession?: string | string[], format?: FileFormat) => {
+    const url = joinUrl(proteinsApiPrefix, 'coordinates');
+    if (accession) {
+      if (Array.isArray(accession)) {
+        return stringifyUrl(url, { accession });
+      }
+      return format
+        ? `${joinUrl(url, accession)}.${fileFormatToUrlParameter[format]}`
+        : joinUrl(url, accession);
+    }
+    return url;
+  },
+  variation: (accession: string, format?: FileFormat) => {
+    const url = joinUrl(proteinsApiPrefix, 'variation', accession);
+    return format ? `${url}.${fileFormatToUrlParameter[format]}` : url;
+  },
   proteins: (accession: string) =>
     joinUrl(proteinsApiPrefix, 'proteins', accession),
-  proteomicsPtm: (accession: string) =>
-    joinUrl(proteinsApiPrefix, 'proteomics-ptm', accession),
+  features: (accession: string, format?: FileFormat, types?: string[]) => {
+    const url = joinUrl(proteinsApiPrefix, 'features', accession);
+    if (types) {
+      return `${url}?types=${types.join(',')}${
```

Comment:
I think the url formation code can be handled by `stringifyUrl` (which calls `stringifyQuery`) from `src/shared/utils/url.ts`.